digital_image_processing/convert_to_grayscale.py

```diff
@@ -0,0 +1,36 @@
+"""
+    Implemented an algorithm using opencv to convert a colored image into its grayscale
+"""
+import doctest
+
+import numpy as np
+from cv2 import destroyAllWindows, imread, imshow, waitKey
+
+
+def convert_to_grayscale(img: np.ndarray) -> np.ndarray:
+    # getting number of pixels in the image
```

Comment:
add below doctest for `convert-to_grayscale` function

        """ 
        >>> img = np.array([[[255, 0, 0], [0, 255, 0]], [[0, 0, 255], [255, 255, 255]]], dtype=np.uint8)
        >>> convert_to_grayscale(img)
        array([[[ 29,  29,  29],
                [151, 151, 151]],

               [[ 76,  76,  76],
                [255, 255, 255]]], dtype=uint8)

        >>> img = np.array([[[255, 255, 255], [0, 0, 0]], [[128, 128, 128], [192, 192, 192]]], dtype=np.uint8)
        >>> convert_to_grayscale(img)
        array([[[255, 255, 255],
                [  0,   0,   0]],

               [[128, 128, 128],
                [192, 192, 192]]], dtype=uint8)
        """